### DISCUSSION TITLE
Details about the training data of BT3/BT4?

### DISCUSSION THREAD

**Discussion by jiacheng-ye** [LeelaChessZero/lc0#1981] [Q&A]
Hi there, thanks for the great work! Based on the recent [blog](https://lczero.org/blog/2024/02/how-well-do-lc0-networks-compare-to-the-greatest-transformer-network-from-deepmind/), I'm curious about the data used when training BT3 and BT4. Are they the same as in the DeepMind paper (i.e., 10M games reannotated by stockfish 16), and whether they are available?

**Comment by borg323** [comment]
BT3 and BT4 were trained using data generated by our training pipeline for T80. I'm not sure of the exact game count, it was using what was created over about 5 months. Our latest training data are in <https://storage.lczero.org/files/training_data/test80/> and their format is described in <https://github.com/LeelaChessZero/lc0/blob/master/src/trainingdata/trainingdata.h>. For mode details join our chat: <https://discord.gg/pKujYxD>

**Reply by 888luck8** [reply]
hey any planning into beating stockfish anytime soon ?
